Document steps for deleting kubectl on macOS (#41300)
kubernetes/website: content/en/docs/tasks/tools/install-kubectl-macos.md

   rm kubectl-convert kubectl-convert.sha256
   ```

> ### Deletion of kubectl on macOS

Reviewer: ```suggestion
### Uninstall kubectl on macOS
```

Since the main task is "install", change this to "uninstall"

> The following methods exist for deleting ```kubectl``` on macOS:

Reviewer: ```suggestion
Depending on how you installed kubectl, use one of the following methods.
```

> 

Reviewer: ```suggestion
### Uninstall kubectl using the command-line
```

> Type in the below command to check where kubectl is installed on your macOS:

Reviewer: ```suggestion
1. Locate the kubectl binary on your system:
```

```bash
where kubectl
```

> (Usually the kubectl installation guides to move the ```kubectl``` binary to ```/usr/local/bin/``` , like ```/usr/local/bin/kubectl```)

Reviewer: ```suggestion

```

Not needed, the where command will locate it anyway

> See the outputs of the command and type in the below command to remove it 

Reviewer: ```suggestion
1. Remove the kubectl binary:
```

```bash

> sudo rm <path> ##replace path with that of the output , eg : sudo sudo rm /usr/local/bin/kubectl

Reviewer: ```suggestion
sudo rm <path> 
```

```

> 

Reviewer: Replace <path> with the path to the kubectl binary from the previous step. For example, `sudo rm /usr/local/bin/kubectl`.

Reviewer 2: I think `whereis  <argument>` the right command rather than `where`.

Reviewer 2: And we can mention this command in one like - 

```bash
sudo rm $(whereis kubectl)
```

Reviewer 2: ```bash
whereis kubectl
```

seems to out more than one path for kubectl (ubuntu 22.04lts) - 
```
kubectl: /usr/bin/kubectl /usr/local/bin/kubectl
```

Instead we can mention - 

```bash
which kubectl
```

o/p -->> `/usr/local/bin/kubectl`

Author: where  works . Are you finding any issues with that @utkarsh-singh1 ?

> If you have installed ```kubectl``` on macOS using ```brew``` , you can just type in the below command to remove it.

Reviewer: ```suggestion
### Uninstall kubectl using homebrew

If you installed kubectl using homebrew, run the following command:
```

Author: Sure @shannonxtreme I will make these changes now itself

   rm kubectl-convert kubectl-convert.sha256
   ```

### Uninstall kubectl on macOS

> Depending on how you installed ```kubectl```, use one of the following methods.

Reviewer: single backticks for inline code please \`kubectl\`

### Uninstall kubectl using the command-line

1. Locate the `kubectl` binary on your system:

> ```bash

Reviewer: Indent this entire code block by three spaces

where kubectl
```

> 2. Remove the `kubectl` binary:

Reviewer: Nit, make the numbering `1.` for every item, the markdown processor will figure out the actual numbers

Author: ohk. I will make the change

> ```bash

Reviewer: Indent this entire code block and line 307 by 3 spaces 

(This is needed because otherwise the site generator thinks these are two separate lists instead of one single list)

> 2. Remove the ```kubectl``` binary:

Reviewer: ```suggestion
1. Remove the `kubectl` binary:
```

sudo rm <path>
```

> Replace with the path to the ```kubectl``` binary from the previous step. For example, ```sudo rm /usr/local/bin/kubectl```

Reviewer: ```suggestion
Replace `<path>` with the path to the `kubectl` binary from the previous step. For example, `sudo rm /usr/local/bin/kubectl`.
```

Reviewer: Single backticks plus telling them which thing to replace (I missed it in my last suggestion)

### Uninstall kubectl using homebrew

> If you installed ```kubectl``` using ```homebrew```, run the following command:

Reviewer: ```suggestion
If you installed `kubectl` using Homebrew, run the following command:
```
